nncf/hw_configs/vpu.json

```diff
@@ -310,6 +310,23 @@
         {"type": "Crop"},
         {"type": "Transpose"},
         {"type": "Tile"},
-        {"type": "StridedSlice"}
+        {"type": "StridedSlice"},
+        // NNCF-specific extensions are below:
+        {
+            "type": "Embedding",
+            "quantization": {
```

Comment:
Shouldn't we explicitly specify here?: 
```
"scales": "unified"
```

Reply:
Well, that depends on our convention, since we basically own the format of our own HW configs now. Everywhere else "scales": "unified" is applied to multi-input ops and refers to the unification of weights between the inputs specifically. The embeddings, instead, have their weights unified, and only in specific conditions when they get "linked" by propagating activation quantizers of downstream real unified scales operations. Anyway I think that specifying this as an option would be probably useful